tests/Directory.Build.props

```diff
@@ -2,7 +2,6 @@
     <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../src'))" />
 
     <ItemGroup>
-        <Compile Include="$(MSBuildProjectDirectory)/../Traits.cs" Link="Properties/Traits.cs"/>
```

Comment:
The pipeline seems not to be running unit tests. IDK if it is related to this change:

```
No test matches the given testcase filter `Category=Unit` in /home/runner/work/loadshedding/loadshedding/tests/unit-tests/Farfetch.LoadShedding.Tests/bin/Release/net8.0/Farfetch.LoadShedding.Tests.dll
```

Reply:
Good catch, fixed. could you check now? thanks

https://github.com/Farfetch/loadshedding/compare/081760d3fdfa6ce6d0c482b076cdd09dabd585ea..e790ff9df75193b41757d2c67da8ba6c8e2d970e

Reply:
All good. Thanks